moma_demos/piloting_demo/launch/sim.launch

```diff
@@ -83,27 +88,18 @@
         <arg name="world_name"  value="$(arg world_name)"/>
     </include>
 
-  <!-- Customer camera info publisher as per real robot -->
-  <arg name="intrinsics_file" value="$(find moma_description)/config/intrinsics_sim.yaml"/>
-  <arg name="camera_info_topic" value="/hand_eye/color/camera_info"/>
-  <node name="camera_intrinsics_publisher" 
-        pkg="moma_sensor_tools" 
-        type="yaml_to_camera_info_publisher.py" 
-        args="$(arg intrinsics_file) $(arg camera_info_topic)" output="screen"/>
-
-
   <!-- Perception -->
-  <include file="$(find piloting_demo)/launch/perception.launch">
-    <arg name="play_bag"          value="false"/>
-    <arg name="base_frame"        value="tracking_camera_odom"/>
-    <arg name="camera_frame"      value="hand_eye_color_optical_frame"/>
-    <arg name="image_topic"       value="/hand_eye/color/image_raw"/>
-    <arg name="depth_topic"       value="/hand_eye/depth/image_rect_raw"/>
-    <arg name="calibration_topic" value="/camera_info"/>
-    <arg name="calibration_file"  value="none"/>
-    <arg name="custom_intrinsics" value="false"/>
-    <arg name="rviz"              value="false"/>
-  </include>
+  <!-- <include file="$(find piloting_demo)/launch/perception.launch">
+    <arg name="sim"                     value="true"/>
```

Comment:
Why no more sim=true? And no more custom_intrinsics?

Reply:
I think I was testing with the ManipulationFromUrdf and I did not need that. It was a temporary change that indeed I had to revert.